src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink_de.properties

```diff
@@ -65,17 +65,19 @@ main.extended.help.footer=Bei Optionen, die eine <Musterliste> erfordern, ist de
 main.runtime.image.linking.cap.enabled=aktiviert
 main.runtime.image.linking.cap.disabled=deaktiviert
 main.runtime.image.linking.cap.sect.header=Funktionen:
-main.runtime.image.linking.cap.msg=\      Assemblierung von Laufzeitimage {0}
+main.runtime.image.linking.cap.msg=\      Verknüpfung von Laufzeitimage {0}
 
 error.prefix=Fehler:
 warn.prefix=Warnung:
 
-err.runtime.link.not.linkable.runtime=Dieses JDK unterstützt keine Assemblierung vom aktuellen Laufzeitimage
-err.runtime.link.jdk.jlink.prohibited=Dieses JDK enthält keine als Pakete verpackten Module und kann nicht verwendet werden, um ein anderes Image mit dem Modul jdk.jlink zu erstellen
-err.runtime.link.packaged.mods=Dieses JDK enthält keine als Pakete verpackten Module. "--keep-packaged-modules" wird nicht unterstützt
-err.runtime.link.modified.file={0} wurde modifiziert
-err.runtime.link.patched.module=Datei {0} nicht im Modulimage gefunden. "--patch-module" wird beim Verknüpfen aus dem Laufzeitimage nicht unterstützt
-err.empty.module.path=leerer Modulpfad
+err.runtime.link.not.linkable.runtime=Dieses JDK unterstützt keine Verknüpfung vom aktuellen Laufzeitimage
+err.runtime.link.jdk.jlink.prohibited=Dieses JDK enthält keine verpackten Module und kann nicht verwendet werden, um ein anderes Image mit dem Modul jdk.jlink zu erstellen
+err.runtime.link.packaged.mods=Dieses JDK enthält keine verpackten Module. "--keep-packaged-modules" wird nicht unterstützt
+err.runtime.link.modified.file={0} wurde geändert
+err.runtime.link.patched.module=jlink unterstützt keine Verknüpfung vom Laufzeitimage unter einer gepatchten Laufzeit mit --patch-module
+err.no.module.path=--module-path-Option muss mit --add-modules ALL-MODULE-PATH angegeben werden
+err.empty.module.path=Kein Modul im Modulpfad "{0}" mit --add-modules ALL-MODULE-PATH gefunden
+err.limit.modules=--limit-modules nicht mit --add-modules ALL-MODULE-PATH zulässig
```

Comment:
I see German has more being changed when compared to Japanese & Chinese. Are these updated translations for German specifically? I see no related change for the other 2 languages for say `err.runtime.link.not.linkable.runtime`.

Reply:
Yes, see the other comment.